Generals/Code/GameEngineDevice/Source/W3DDevice/GameClient/W3DAssetManager.cpp

```diff
@@ -787,12 +761,8 @@ RenderObjClass * W3DAssetManager::Create_Render_Obj(
 			StringClass	new_filename = StringClass("..\\") + filename;
 			if (Load_3D_Assets( new_filename ) == false)
 			{
-#ifdef	INCLUDE_GRANNY_IN_BUILD
-				char *mesh_name = ::strchr (filename, '.');
-				::lstrcpyn (mesh_name, ".gr2",5);
-				Load_3D_Assets( filename );
-				isGranny=true;
-#endif
+				// TheSuperHackers @info The asset could not be found
```

Comment:
This info is not necessary. The condition with `Load_3D_Assets( new_filename ) == false` is self explanatory here.